src/main/java/com/minecolonies/coremod/achievements/ModAchievements.java

```diff
@@ -408,6 +396,9 @@ private ModAchievements()
      */
     public static void init()
     {
-        AchievementPage.registerAchievementPage(achievementPageMineColonies);
+        //todo add to an advancement page
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1135)